pkg/service/utils.go

```diff
@@ -22,8 +21,7 @@ var ServiceSet = wire.NewSet(
 )
 
 func handleError(w http.ResponseWriter, status int, msg string) {
-	l := logger.GetLogger().WithOptions(zap.AddCallerSkip(1))
```

Comment:
we'll need the CallerSkip here. otherwise the error would be useless as they'll show the error happening in utils.go/handleError.

Reply:
@davidzhao Good point, will fix